### DISCUSSION TITLE
Empty shape

### DISCUSSION THREAD

**Discussion by LeXXik** [jrouwe/JoltPhysics.js#201] [Q&A] [answered]
Given that an empty shape has no volume and does not collide with anything, what is a common use case when we'd like to use it?

**Comment by jrouwe** [comment, marked as answer]
Does this help?

https://github.com/jrouwe/JoltPhysics/commit/c244674f03cc329c8121dc76c3dc4187c14d0b5b

**Reply by LeXXik** [reply]
Makes sense, thank you!
